project-base/src/Shopsys/ShopBundle/Migrations/Version20190918155540.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\ShopBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+/**
+ * This migration changes the default currencies settings - it should be ran on the clean project-base installation only!
+ */
+class Version20190918155540 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->sql('UPDATE currencies SET exchange_rate = 0.04 WHERE code = \'CZK\'');
+        $this->sql('INSERT INTO currencies (id, name, code, exchange_rate) VALUES (2, \'Euro\', \'EUR\', 1)');
+        $this->sql('UPDATE setting_values SET value = 2 WHERE name = \'defaultCurrencyId\'');
+        $this->sql('UPDATE setting_values SET value = 2 WHERE name = \'defaultDomainCurrencyId\' AND domain_id = 1');
```

Comment:
This migration is OK on project-base. However it is problem for existing projects. 

Why not modify existing migrations?

Yes, I agree with "Migrations can not be modified", but this is nice example for it. In my opinion it does not have any disadvantage :grin: in this case.

Yes, projects can skip this migration. But why, If we can prevent it.

Reply:
:thinking: I think that I am ok with changing the existing migration... :slightly_smiling_face:

Reply:
@pk16011990 has a valid point, can you please update it @vitek-rostislav ?